websites/V/Video Game Mods/presence.ts

```diff
@@ -0,0 +1,156 @@
+const presence = new Presence({
+		clientId: "1079522783974920212",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			details: "Other",
+			largeImageKey: "https://i.imgur.com/zvX0TaN.png",
+			startTimestamp: browsingTimestamp,
+			buttons: [
+				{
+					label: "View Page",
+					url: window.location.href,
```

Comment:
In this repository, `document.location` is preferred over `window.location`
```suggestion
					url: document.location.href,
```